msu/utils/tile.nut

```diff
@@ -12,4 +12,29 @@
 		
 		return true;
 	}
+
+	function getNeighbors( _tile )
+	{
+		local ret = [];
+		for (local i = 0; i < 6; i++)
+		{
+			if (_tile.hasNextTile(i))
+				ret.push(_tile.getNextTile(i));
+		}
+		return ret;
+	}
+
+	function getNeighbor( _tile, _function )
+	{
+		// _function( tile ) and returns boolean
```

Comment:
this needs to be better, doesn't really explain anything and should be using `_tile` regardless. (can be removed though and that would be fine imo.

Reply:
The function is correct. It does the following: "Give me the first neighbor of `_tile` for which `_function` returns true`.

Reply:
I'm not saying the function is wrong, I'm saying the comment isn't useful